outsource/outsource.py

```diff
@@ -312,6 +311,44 @@ def _generate_tc(self):
     def _generate_rc(self):
         return ReplicaCatalog()
 
+    def make_tarballs(self):
+        """Make tarballs of Ax-based packages if they are in editable user-
+        installed mode."""
+        tarballs = []
+        tarball_paths = []
+        for package_name in ["strax", "straxen", "cutax"]:
```

Comment:
Maybe we should consider moving this hardcoded packages to something tracked in xenon_config. Good for now. I will start an issue and hopefully get new people onto it.

Reply:
Oh wait there is already an https://github.com/XENONnT/outsource/issues/74.